plugins/tag_search/sphinx/kTagFlowManager.php

```diff
@@ -266,16 +290,16 @@ private static function updateInstanceCountList(array $foundTagObjects, $time, $
             $connection = Propel::getConnection();
             foreach ($foundTagObjects as $tag)
             {
-	            $foundTagIds[]=$tag->getId();
+	            $foundTagIds[] = $tag->getId();
             }
-            $idsString= "(".implode( ",",$foundTagIds).")";
-            if($toIncrement==true)
+            $idsString = "(" . implode( ",",$foundTagIds) . ")";
+            if($toIncrement == true)
             {
-                $updateSql = "update tag set instance_count=instance_count+1 , updated_at = '".$time."' where id in ".$idsString.";";
+                $updateSql = "update tag set instance_count=instance_count+1, updated_at=$time where id in $idsString;";
```

Comment:
why the following statements are done directly on the db?

Reply:
@inbal-ben-david that's a great question that can be addressed to the original author of this code.